test/BulletDynamics/test_deterministic_save_and_restore.cpp

```diff
@@ -0,0 +1,221 @@
+#include "../../Extras/Serialize/BulletFileLoader/bFile.h"
+#include <btBulletDynamicsCommon.h>
+#include "../../src/BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.h"
+#include "../../src/BulletDynamics/Featherstone/btMultiBodyConstraintSolver.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btMultiBodyWorldImporter.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btBulletWorldImporter.h"
+#include "../../Extras/Serialize/BulletWorldImporter/btWorldImporter.h"
+#include "../../src/BulletCollision/CollisionDispatch/btCollisionObject.h"
+#include "../../src/BulletCollision/CollisionDispatch/btCollisionWorld.h"
```

Comment:
I believe files under src/ are in the include path, so you shouldn't need to prefix them with ../../src